build/Targets/Imports.targets

```diff
@@ -9,63 +9,22 @@
   <Import Project="RepoToolset\Workarounds.targets"/>
   <Import Project="RepoToolset\OptimizationData.targets"/>
   <Import Project="RepoToolset\RepositoryInfo.targets"/>
+  <Import Project="RepoToolset\Localization.targets" />
+  <Import Project="RepoToolset\StrongName.targets" />
+  <Import Project="RepoToolset\SymStore.targets" />
+  <Import Project="RepoToolset\GenerateInternalsVisibleTo.targets" />
   <Import Project="VisualStudio.targets"/>
+  <Import Project="Roslyn.Toolsets.Xunit.targets" Condition="'$(_IsAnyUnitTest)' == 'true'" />
```

Comment:
>_IsAnyUnitTest [](start = 64, length = 14)

Given this property is used across targets files should we be using the `_` prefix? My understanding is that the convention is to only use that for private values within a single file.

Reply:
Yes, I'll replace this property with `IsTestProject` in follow-up PR.